src/pages/frequently.js

```diff
@@ -0,0 +1,62 @@
+import Button from "../components/button";
```

Comment:
just a nit: `faq` would be a better file name ;)

Reply:
fixed...